src/Entities.js

```diff
@@ -17,6 +17,31 @@ limitations under the License.
 var React = require('react');
 var sdk = require('./index');
 
+function isMatch(query, name, uid) {
+    query = query.toLowerCase();
+    name = name.toLowerCase();
+    uid = uid.toLowerCase();
+
+    // direct prefix matches
+    if (name.indexOf(query) === 0 || uid.indexOf(query) === 0) {
+        return true;
+    }
+
+    // strip @ on uid and try matching again
+    if (uid.length > 1 && uid.substring(1).indexOf(query) === 0) {
```

Comment:
might be nice to match the comment and avoid magic and only strip if first char is @